pkg/validate/validate_test.go

```diff
@@ -0,0 +1,14 @@
+package validate
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetSchemaLocations(t *testing.T) {
+	schemaLocations := getSchemaLocations()
+
+	// default schema location is "./schemas"
+	assert.Equal(t, "./schemas/{{ .NormalizedKubernetesVersion }}/{{ .ResourceKind }}{{ .KindSuffix }}.json", schemaLocations[0])
+}
```

Comment:
I thought of this, but I wasn't sure if we want to clone repos during testing. I could add development overhead, since tilt will always run tests before starting server.